conans/test/integration/system_reqs_test.py

```diff
@@ -139,3 +144,45 @@ def wrong_output_test(self):
         pref = PackageReference(ref, "f0ba3ca2c218df4a877080ba99b65834b9413798")
         load_file = load(client.cache.system_reqs_package(pref))
         self.assertEqual('', load_file)
+
+    def remove_system_reqs_test(self):
+        ref = ConanFileReference.loads("Test/0.1@user/channel")
+        client = TestClient()
+        files = {'conanfile.py': base_conanfile.replace("%GLOBAL%", "")}
+        client.save(files)
+        system_reqs_path = os.path.join(
+            client.cache.package_layout(ref).conan(), SYSTEM_REQS_FOLDER)
+
+        # create package to populate system_reqs folder
+        self.assertFalse(os.path.exists(system_reqs_path))
+        client.run("create . user/channel")
+        self.assertIn("*+Running system requirements+*", client.user_io.out)
+        self.assertTrue(os.path.exists(system_reqs_path))
+
+        # a new build must not remove or re-run
+        client.run("create . user/channel")
+        self.assertNotIn("*+Running system requirements+*", client.user_io.out)
+        self.assertTrue(os.path.exists(system_reqs_path))
+
+        # error must not remove anything
+        with self.assertRaisesRegexp(
+                Exception, "ERROR: Please specify a valid package reference to be cleaned"):
+            client.run("remove --system-reqs")
+        self.assertTrue(os.path.exists(system_reqs_path))
+
+        # error must show exception message
+        with self.assertRaises(Exception) as error:
+            client.run("remove --system-reqs foo/bar@foo/bar")
+            self.assertIn("ERROR: Could not remove system_reqs: [Errno 2] No such file or directory:", error.exception)
```

Comment:
I'm not sure the user is expecting this to happen. Probably only if something weird happens like permissions or whatever. If the recipe doesn't exist it should fail with that message. Otherwise if the recipe exists but there is no cached system requirements it should exit normally.

Reply:
so, should we ignore errors, as made to `--locks` ?

Reply:
Done, errors will be ignored.

Reply:
why ignore? permission errors happen, especially on Windows (e.g. due to the AV software, search indexers, etc.). as a user, I would prefer commands to report if they weren't success, and return correct exit code in that case.
but we should clearly distinguish cases of missing recipe vs permission issues for sure.

Reply:
If we change this behavior for system-reqs, I recommend do the same for `--locks`

Reply:
As @SSE4 said I don't want to ignore the failures. But if a recipe doesn't exist it should fail by saying "the recipe doesn't exist", not a weird error. And if there is no system requirements folder for a recipe that exists, it should end with ok. (I think that test is missing)

Reply:
understood

Reply:
@SSE4 @lasote  done! now it will raises friendly error messages.